bin/GameIndex.dbf

```diff
@@ -9946,15 +9946,9 @@ Name   = Sega Bass Fishing Duel
 Region = PAL-M5
 ---------------------------------------------
 Serial = SLES-51390
-Name   = Fightbox
+Name   = FightBox
 Region = PAL-E
 vuClampMode = 0 // Fixes missing 3D
-[patches = E7EFA0BF]
-	comment= Gets around GS SIGNAL hang, game expects value other than C in SIGNAL but the PATH2 packet sends double 0xC
-	comment= The SIGNAL value is compared to the value at address 0x34CDF3, if this address contains 0x4 (OR'd with 0x8)
-	commend= the interrupt doesn't get acknoledged and cleared and so the PATH2 packet hangs
-	patch=1,EE,001f5a0c,word,30840008
-[/patches]
```

Comment:
Why was this removed?

Reply:
This was already removed in the latest dev builds where ref fixed it, somehow github says it gets deleted again (which is wrong). It's the same as there. Not sure why its saying that about Fightbox

Reply:
A bit odd, maybe rebase with latest master?

Reply:
Huh, it was missing a commit, but it reverted that commit for some reason. Anyway it seem to have being fixed.